src/util.js

```diff
@@ -86,9 +86,16 @@ export function getLessons(lessonContext, readmeContext, courseContext) {
   }, {});
 }
 
-export function getCourseInfo(courseName) {
-  const courseInfo = require('onlyContent!lessonSrc/' + courseName + '/index.md');
-  return courseInfo.content;
+/**
+* Returns /course/index_(ISO_CODE) if it exists, returns /course/index if not.
+**/
+export function getCourseInfo(courseName, language) {
+  try {
+    return require('onlyContent!lessonSrc/' + courseName + '/index_' + language + '.md').content;
+  }
+  catch(err) {
+    return require('onlyContent!lessonSrc/' + courseName + '/index.md').content;
```

Comment:
I agree that this may not be the optimal solution, but using try catch is the only statement I found where `require(..)` can be used! I can make a work around though :+1: